stumpy/stump.py

```diff
@@ -116,15 +121,31 @@ def _compute_diagonal(
         The thread index
 
     ρ : numpy.ndarray
-        The Pearson correlations
+        The (top-k) Pearson correlations, sorted in ascending order per row
 
     I : numpy.ndarray
-        The matrix profile indices
+        The (top-k) matrix profile indices
+
+    ρL : numpy.ndarray
+        The top-1 left Pearson correlations
+
+    IL : numpy.ndarray
+        The top-1 left matrix profile indices
+
+    ρR : numpy.ndarray
+        The top-1 right Pearson correlations
+
+    IR : numpy.ndarray
+        The top-1 right matrix profile indices
 
     ignore_trivial : bool
         Set to `True` if this is a self-join. Otherwise, for AB-join, set this to
         `False`. Default is `True`.
 
+    k : int
+        The number of smallest elements in distance profile that should be stored
```

Comment:
> Note that this will increase the total computational time and memory usage when k > 1."

Ahh, I forgot to add this in docstring in spite of you mentioning that before. Sorry about that.

I will use the proposed definition. Thanks!